packages/service-clients/odsp-client/src/interfaces.ts

```diff
@@ -27,7 +27,7 @@ export interface OdspConnectionConfig {
 	siteUrl: string;
 
 	/**
-	 * RaaS Drive Id of the tenant where Fluid containers are created
+	 * SharePoint Embedded Container Id of the tenant where Fluid containers are created
```

Comment:
`SPE drive id` is container id in SPE terms. All the SPE docs mention container id and not drive id. I wanted to match what the doc says to avoid confusion.

Reply:
Sounds good, thank you for confirming!